src/app/pages/viewing/responses-list/responses-list.component.html

```diff
@@ -1,70 +1,32 @@
-<!-- Desktop response-list format -->
-<div class="response-list-card d-none d-sm-none d-md-none d-lg-block mt-4">
-  <div class="card p-3 border-0">
-    <div class="row user-info">
-      <div class="col-lg-3 col-md-4">
-        <div class="profile-pic-container">
-          <img class="profile-pic" src="{{ photoUrl }}" alt="profile-pic">
-        </div>
-      </div>
-      <div class="col-lg-9 col-md-8">
-        <label class="mt-2 username">{{ displayName }}</label>
-      </div>
-    </div>
-    <div class="dropdown-divider"></div>
-    <ul class="nav flex-column ml-2">
-      <li class="nav-item mb-2 response-title">
-        My Responses
-      </li>
-    </ul>
-    <div id="author-list">
-      <ul class="nav flex-column ml-2">
-        <!--suppress HtmlUnknownAttribute -->
-        <li class="nav-item mb-1" *ngFor="let msg of viewService.messages; let i = index" data-target="#carousel"
-            attr.data-slide-to="{{ i + 1 }}" (click)="showMessage(msg.id);">
-          <a href="javascript: void(0)">{{ msg.fromName }}</a>
-        </li>
-      </ul>
-    </div>
-  </div>
+<div class="hamburger vertical-center shadow" (click)="navOpen(true)">
+  <img src="../../../../assets/images/icons/chevron-right.svg">
```

Comment:
Missing required alt attribute
```suggestion
  <img src="../../../../assets/images/icons/chevron-right.svg" alt="Open list of messages">
```